.travis.yml

```diff
@@ -147,55 +164,29 @@ matrix:
         postgresql: "9.6"
       env:
         - PG_VERSION=9.6
-        - TEST_CLIENTS=Y
-    - jdk: oraclejdk8
-      addons:
-        postgresql: "9.6"
-      env:
-        - PG_VERSION=9.6
         - QUERY_MODE=simple
         - COVERAGE=Y
         - ANORM_SBT=Y
     - jdk: oraclejdk8
       addons:
-        postgresql: "9.6"
+        postgresql: "9.3"
```

Comment:
what is the rationale of switching to 9.3 here ?

Reply:
I have dropped 2 jobs that target openjdk 6 and 7 (that the part of streamlining), and one of those jobs test on 9.3, which I wish to keep testing as there are many 9.6.